packages/client/src/components/OrderReview/OrderReview.jsx

```diff
@@ -83,7 +84,11 @@ export const OrderReview = ({ movies }) => {
                   </ItemsCardTypographyDescription>
                   <ItemFavRemoveBox>
                     <FavRemoveBox>
-                      <FavRemoveButton onClick={(e) => e.preventDefault()}>
+                      <FavRemoveButton
+                        onClick={() => {
+                          handleAddFavorite(movie);
```

Comment:
nothing wrong with the logic, but the naming is slightly off for me: remove button handles adding? I would suggest to adjust the naming a little so it's less confusing :)

Reply:
ahh yeah i did for the same since they have same style but yeah now when i look it is a bit confusing but i'll fx it with other branch coming soon :D